src/styles/pages/blog.scss

```diff
@@ -44,12 +44,12 @@
 		height: auto;
 		border: 1px solid var(--surface-stroke-default);
 		border-radius: var(--overlay-corner-radius);
-		box-shadow: 0 2.74416px 2.74416px hsl(0, 0%, 0% / 3%),
-		0 5.48831px 5.48831px hsl(0, 0%, 0% / 4%),
-		0 13.7208px 10.9766px hsl(0, 0%, 0% / 5%),
-		0 20.5812px 20.5812px hsl(0, 0%, 0% / 6%),
-		0 41.1623px 41.1623px hsl(0, 0%, 0% / 7%),
-		0 96.0454px 89.1851px hsl(0, 0%, 0% / 9%);
+		box-shadow: 0 2.74416px 2.74416px hsl(0, 0%, 0%, 3%),
+		0 5.48831px 5.48831px hsl(0, 0%, 0%, 4%),
+		0 13.7208px 10.9766px hsl(0, 0%, 0%, 5%),
+		0 20.5812px 20.5812px hsl(0, 0%, 0%, 6%),
+		0 41.1623px 41.1623px hsl(0, 0%, 0%, 7%),
+		0 96.0454px 89.1851px hsl(0, 0%, 0%, 9%);
```

Comment:
Yes. Sass confuses `/` to be the division operator instead of what it actually is - a CSS separator, so it divides the intended lightness value by the alpha value, making for a much darker color.

Hoewever, `,` is unambiguous, so it works fine.